src/database/FirehoseConnector.js

```diff
@@ -161,7 +167,11 @@ class FirehoseConnector {
                 totalPacketsSent,
                 totalReceivedPacketsLost,
                 totalSentPacketsLost,
-                meanRtt
+                meanRtt,
+                meanPrincipalFrameHeight: principalVideoAggregates.meanFrameHeight,
+                meanPrincipalFramesPerSecond: principalVideoAggregates.meanFramesPerSecond,
+                meanSecondaryFrameHeight: secondaryVideoAggregates.meanFrameHeight,
+                meanSecondaryFramesPerSecond: secondaryVideoAggregates.meanFramesPerSecond
```

Comment:
The average principal/secondary frame rate/resolution is a new peer-connection level quality measurement statistic. Do you guys agree to add these in the quality metrics table in redshift?